nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -354,9 +354,21 @@ php.override {
     </para>
    </listitem>
    <listitem>
-   <para>
-     The default output of <literal>buildGoPackage</literal> is now <literal>$out</literal> instead of <literal>$bin</literal>.
-   </para>
+    <para>
+      The default output of <literal>buildGoPackage</literal> is now <literal>$out</literal> instead of <literal>$bin</literal>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Packages built using <literal>buildRustPackage</literal> now use the <literal>release</literal>
+     mode for the <literal>checkPhase</literal> by default.
+    </para>
+    <para>
+     Please note that <literal>rust</literal>-packages utilizing a custom build/install-procedure
+     (e.g. by using a <filename>Makefile</filename>) or test-suites that rely on the
+     structure in the <filename>target/</filename>-directory may break because of that.
+     For further information, please read the rust-section in the nixpkgs manual.
```

Comment:
I don't see why `rust` needs to be a literal, rather than just using "Rust".

```suggestion
     Please note that Rust packages utilizing a custom build/install procedure
     (e.g. by using a <filename>Makefile</filename>) or test suites that rely on the
     structure of the <filename>target/</filename> directory may break due to those assumptions.
     For further information, please read the Rust section in the Nixpkgs manual.
```